change/@fluentui-react-examples-583178b2-dbbd-4bc6-b09e-2c324391cfd6.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
This should technically be patch since it affects a published example (though practically it doesn't matter since the example package shouldn't be consumed by anything but internal tooling experiments)

Reply:
Sure, will update! I wasn't sure when running change which of those it should be 😄